worker/src/RTC/SimpleConsumer.cpp

```diff
@@ -328,13 +331,13 @@ namespace RTC
 		}
 
 		// Process the packet.
-		if (this->rtpStream->ReceivePacket(packet))
+		if (this->rtpStream->ReceivePacket(sharedPacket ? nullptr : packet, sharedPacket))
```

Comment:
Will sharedPacket match null of empty? I think so but just in case.

Reply:
Both. It will  match if the `sharedPacket` was not given by argument or if the sharedPacket is given but has no RtpPacket pointer in it.